kolibri/core/assets/src/views/PaginatedListContainer.vue

```diff
@@ -5,19 +5,13 @@
       <KGridItem :layout12="{ span: 7 }">
         <slot name="otherFilter"></slot>
       </KGridItem>
-      <KGridItem
-        :layout12="{ span: 5, alignment: 'right' }"
-        class="text-filter"
-      >
-        <FilterTextbox
-          v-model="filterInput"
-          :placeholder="filterPlaceholder"
-        />
+      <KGridItem :layout12="{ span: 5, alignment: 'right' }" class="text-filter">
```

Comment:
this indentation change does not follow our [frontend code conventions](https://kolibri-dev.readthedocs.io/en/develop/frontend_architecture/conventions.html)